oarepo_doi/services/provider.py

```diff
@@ -313,6 +429,27 @@ def community_slug_for_credentials(self, value):
             return c._source.slug
         except:
             return value
+    def get_versions(self, record):
+        topic_service = get_record_service_for_record(record)
+        versions = topic_service.search_versions(system_identity, record.pid.pid_value)
```

Comment:
note: we should pass sufficiently large params={'size': 1000} or use pagination here as the search_versions will be paginated with default of 25 versions.